spec/features/providers/show_spec.rb

```diff
@@ -18,6 +18,11 @@
       visit provider_path(provider.provider_code)
       expect { organisation_show_page.courses_as_accredited_body_link }.to raise_error(Capybara::ElementNotFound)
     end
+
+    it "does not have the request PE courses for 2021/22 link" do
+      visit provider_path(provider.provider_code)
+      expect { organisation_show_page.request_allocations_link }.to raise_error(Capybara::ElementNotFound)
```

Comment:
using exception as a verification mechanism is a bad test

https://www.rubydoc.info/gems/site_prism/frames#Testing_that_an_element_does_not_exist

Reply:
I agree, was just following the pattern in this file. I will change them all.